spec.html

```diff
@@ -35285,9 +36352,46 @@ <h1>CloneArrayBuffer ( _srcBuffer_, _srcByteOffset_, _srcLength_, _cloneConstruc
         </emu-alg>
       </emu-clause>
 
-      <emu-clause id="sec-rawbytestonumber" aoid="RawBytesToNumber">
-        <h1>RawBytesToNumber ( _type_, _rawBytes_, _isLittleEndian_ )</h1>
-        <p>The abstract operation RawBytesToNumber takes three parameters, a String _type_, a List _rawBytes_, and a Boolean _isLittleEndian_. This operation performs the following steps:</p>
+      <emu-clause id="sec-isunsignedelementtype" aoid="IsUnsignedElementType">
+        <h1>IsUnsignedElementType ( _type_ )</h1>
+        <p>The abstract operation IsUnsignedElementType verifies if the argument _type_ is an unsigned TypedArray element type. This operation performs the following steps:</p>
+        <emu-alg>
+          1. If the first code unit of _type_ is the code unit 0x0055 (LATIN CAPITAL LETTER U) or _type_ is `"BigUint64"`, return *true*.
```

Comment:
I would prefer something explicit rather than relying on the first letter being "U".

Reply:
Done.